CMakeLists.txt

```diff
@@ -346,12 +353,19 @@ if(UNIX AND NOT APPLE AND NOT ANDROID)
 
 	if(USE_GLES2)
 		target_compile_definitions(${PROJECT_NAME} PRIVATE GLES GLES2)
-		target_link_libraries(${PROJECT_NAME} PRIVATE GLESv2)
+		if(USE_VIDEOCORE)
+			target_link_libraries(${PROJECT_NAME} PRIVATE "-lbrcmGLESv2")
+			target_link_directories(${PROJECT_NAME} PRIVATE "/opt/vc/lib")
+		else()
+			target_link_libraries(${PROJECT_NAME} PRIVATE GLESv2)
+		endif()
 	elseif(USE_GLES)
 		target_compile_definitions(${PROJECT_NAME} PRIVATE GLES GLES3)
 		if(NOT SDL2_FOUND)
 			target_link_libraries(${PROJECT_NAME} PRIVATE EGL)
 		endif()
+		find_library(${GLES3_LIBRARIES} NAMES GLESv3 GLESv2 NO_CACHE REQUIRED)
```

Comment:
@cmitu `${GLES3_LIBRARIES}` is not right here, is it ? Shouldn't it be `GLES3_LIBRARIES` like in line 98 ?

Reply:
Yes, thank you for the correction. Should be fixed by #449, sorry for not noticing it before submitting the PR.